QUEST.tsv

```diff
@@ -45,251 +45,251 @@ QUEST_20150317_000044	Items to be deleted
 QUEST_20150317_000190	What's up?
 QUEST_20150317_000191	Really?
 QUEST_20150317_000192	Pond
 QUEST_20150317_000193	What happened?
 QUEST_20150317_000194	Gesti told us to protect this place.
-QUEST_20150317_000195	So, what should we do now?
+QUEST_20150317_000195	So what should we do now?
 QUEST_20150317_000196	Is this the right place?
 QUEST_20150317_000197	Explaining
-QUEST_20150317_000198	Shared the details of the revelation fragment found in the Crystal Mines.
-QUEST_20150317_000199	It doesn't matter how shiny your past is, as long as you follow the Royal Army's orders. {nl}Same for the Kaliss Knights. Royal decree makes us the law in here.
+QUEST_20150317_000198	Shared the details of the revelation fragment found in the Crystal Mine.
+QUEST_20150317_000199	I don't know how great you were in the past nor do I care. {nl}In here, you follow the Royal Army's orders, even you Kaliss Knights. {nl}In here, our word is law!
 QUEST_20150317_000200	Graverobber Hubertas
-QUEST_20150317_000201	Hanging around these ruined places is driving me insane. {nl}Why the hell did he come here? 
+QUEST_20150317_000201	Hanging around in these ruins is driving me insane. {nl}Why the heck did he come here anyway?
 QUEST_20150317_000202	Graverobber Benjaminas
-QUEST_20150317_000203	People say we're grave robbers, but we prefer to call ourselves…archaeologists. {nl}With all this valuable stuff left around, I'd say the Goddess wanted us to take it. {nl}It sure beats leaving it all just to rot, right?
+QUEST_20150317_000203	People call us grave robbers, but we prefer to call ourselves… archaeologists, {nl}scavenging and making use of all these items no one uses anymore. {nl}It sure beats leaving them to rot, right?
 QUEST_20150317_000204	A Citizen's Journal
-QUEST_20150317_000205	It's been a year since the war broke out. Prices in town have risen. {nl}General Ruklys has sent for aid, to restore the town's dwindling supplies. {nl}It's been a while since any good news. There's talk about King Kadumel aiming at Ruklys. {nl}Mood's not going to change soon.
+QUEST_20150317_000205	It's been a year since the civil war broke out. Prices in town have risen dramatically. {nl}General Ruklys has sent for aid, to restore the town's dwindling supplies. {nl}It's been a while since we've had any good news. There's talk about King Kadumel going after the general. {nl}Mood's not going to change soon.
 QUEST_20150317_000206	A Sentry's Report
-QUEST_20150317_000207	Security levels were raised in response to King Kadumel's forces and the pressing war situation. {nl}The clumsy king is commanding a large force. {nl}If General Hayatin can bring enough volunteers, Kadumel might surrender or retreat.
+QUEST_20150317_000207	The guard posts have been reinforced in response to the king's forces movements in the country. {nl}It's been confirmed that the king commands a large army. {nl}But if General Hayatin can muster enough volunteer soldiers, the incompetent king will likely surrender or retreat.
```

Comment:
Would it make sense here to keep the mention of "King Kadumel" instead of "the king" the first time, then refer to "the king" afterwards?